act-framework.bs

```diff
@@ -53,51 +53,45 @@ ACT Input Data {#input}
 
 Test Input Types {#input-types}
 -------------------------------
-Web pages, including publications and applications go through many different stages before it is rendered to the end user. For example, PHP may be used to put various pieces of content into a template. That template is then sent as an HTML text document to a web browser, which in term parses it and turns it into a DOM tree, before rendering it to the screen. At each of these stages, accessibility tests could be run. We can divide this into two stages of testing, black-box and white-box testing.
 
-### Black-box testing
+Web pages, including publications and applications go through many different stages before it is rendered to the end user. For example, PHP may be used to put various pieces of content into a template. That template is then sent as an HTML text document to a web browser, which in term parses it and turns it into a DOM tree, before rendering it to the screen. At each of these stages, accessibility tests could be run. It is therefore important to specify the *type of input* that an ACT Rule expects.
 
-In black box testing the final product is tested, without looking at the code that generated it. To do black-box testing, the tester does not need access to the source file(s), which usually makes black-box testing easier to set up.
+The following input types are common in accessibility testing:
 
-There are different types of black-box testing:
-- File testing
-- DOM tree testing
-- Browser testing
-- Driver testing
+- HTTP Response
+- DOM Tree
+- Rendered Page
+- (Web)Driver
```

Comment:
It seems to me that this is not really of the same nature as the 3 bullets above. 
The previous items describe **content** while a Web Driver is more an API. Eventually, the driver returns a view of the content (typically a DOM tree), which is what is tested.

Maybe be "Actionable page" better describes what is meant here?